app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/model/VpnDatabaseModels.kt

```diff
@@ -73,9 +67,8 @@ data class AlwaysOnState(val alwaysOnEnabled: Boolean, val alwaysOnLockedDown: B
 
 @Entity(tableName = "vpn_service_state_stats")
 data class VpnServiceStateStats(
-    @PrimaryKey(autoGenerate = true) val id: Int = 0,
-    val timestamp: String = DatabaseDateFormatter.timestamp(),
-    val state: VpnServiceState,
+    val timestamp: Long = System.currentTimeMillis(),
```

Comment:
Do you want this to be indexed since you are ordering by it?

Reply:
Only two entries max in the db, we can probably not index it